Documentation/articles/introduction/help_and_support.md

```diff
@@ -8,9 +8,8 @@ If you want to find an answer to a more specific problem, you can ask it on our
 
 You can also chat to the members of the MonoGame community via [Gitter](https://gitter.im/MonoGame/MonoGame) or [Discord](https://discord.gg/xCehJFD) using the links below
 
-| | |
-|--|--|
 |[![Gitter](~/images/third_party/gitterlogo.jpg)](https://gitter.im/MonoGame/MonoGame) |[![Discord](~/images/third_party/discordlogo.png)](https://discord.gg/xCehJFD) |
+|--|--|
```

Comment:
Hah... now the issue moved below the images...

![image](https://user-images.githubusercontent.com/103522/88601360-1bb6d600-d036-11ea-9e63-1958fc644fb0.png)

... i think we just need to stack them and not worry about centering them.

Reply:
Can also use HTML if we want to.

Reply:
OK, will stack them :D  Was hoping that altering the markdown would be enough, but I guess not :D